gentle/resample.py

```diff
@@ -68,7 +68,7 @@ def resample_sox(infile, outfile, offset=None, duration=None):
 def resample(infile, outfile, offset=None, duration=None):
     if not os.path.isfile(infile):
         raise IOError("Not a file: %s" % infile)
-    if shutil.which(FFMPEG):
+    if shutil.which(FFMPEG) or os.path.exists(FFMPEG):
```

Comment:
what's the case where `shutil.which(FFMPEG)` returns `None` but the path exists? wouldn't that correspond to a broken symlink?

Reply:
I found that 'which' doesn't work as expected within a frozen environment. Doing 'which /abs/path/to/ffmpeg' works perfectly fine in a normal file system, but when I tried it after bundling the source it returns empty. I tried googling for explanations but couldn't find anything on this weird behavior. Checking for path existence still works in a frozen environment, however, so I used that as a fallback.

Reply:
weird and interesting! a comment in the code might help others understand what's going on, but I'm also fine to merge this in as is